src/components/Author.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import styled from 'styled-components';
+import { Link } from 'react-router-dom';
+
+const TableLink = styled(Link)`
+  text-decoration: none;
+  color: ${({ theme }) => theme.color.externalLink};
+  font-size: ${({ theme }) => theme.font.size.small};
+`;
+
+const Author = (author) => {
+  return author === '[deleted]' ? (
+    <>author</>
```

Comment:
I guess you meant `<>{author}</>`? :smile: At least that's what you had before

Reply:
yeah... oops! 😆